experimental/traffic-portal/src/app/shared/navigation/navigation.service.ts

```diff
@@ -143,6 +143,12 @@ export class NavigationService {
 				name: "Tenants"
 			}],
 			name: "Users"
+		}, {
+			children: [{
+				href: "/core/statuses",
+				name: "Statuses"
+			}],
+			name: "Configure"
```

Comment:
This is fine, because it's what TPv1 did, but fwiw I'd put it under Servers. If we want to do that it'll be trivial to do later, so it's not a big deal either way.

Reply:
I moved statuses under servers.